tests/OptionGroupTest.cpp

```diff
@@ -421,7 +421,7 @@ struct ManyGroups : public TApp {
         g3->add_option("--val3", val3);
     }
 
-    void remove_required() {
+    void remove_required() const {
```

Comment:
I suspect it doesn't matter but it is kind of misleading for this to be const.

Reply:
Okay, will do that.